static/json-schema-viewer/index.html

```diff
@@ -0,0 +1,238 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>JSON Schema Viewer 0.3.4</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/jquery-mobile/1.4.5/jquery.mobile.min.css" />
+        <link rel="stylesheet" href="styles/json-schema-viewer.min.css" />
+
+    </head>
+    <body>
+        <div id="loading">
+            <div id="preload"><img src="images/loader.gif" alt="Loading..."/>
+                <br />
+                Loading...
+            </div>
+        </div>
+        <div class="ui-responsive-panel" data-role="page" id="viewer-page"  data-title="JSON Schema Viewer">
+            <div data-role="header" class="md-navbar">
+                <div data-role="controlgroup" data-type="horizontal" class="ui-btn-left">
+                    <a class="ui-btn ui-corner-all ui-icon-bars ui-btn-icon-notext" href="#info-panel" title="Toggle Info">Menu</a>
+                    <a class="ui-btn md-flex-btn ui-corner-all ui-icon-check ui-btn-icon-left" href="#validator-page" title="Go to Validator">Validator</a>
+                    <!--<a href="#" class="ui-btn ui-btn-icon-right ui-icon-plus ui-btn-icon-notext ui-corner-all">Zoom-in</a>
+                    <a href="#" class="ui-btn ui-btn-icon-right ui-icon-minus ui-btn-icon-notext ui-corner-all">Zoom-out</a>-->
+
+                </div>
+                <h1 id="viewer-title">JSON Schema Viewer<span class="schema-version"></span></h1>
+                <!-- <a class="ui-btn-right ui-btn ui-icon-info ui-btn-icon-notext ui-corner-all" href="#help-panel"  title="Toggle Help">Help</a> -->
+                <a data-rel="popup" data-role="none" data-position-to="window" class="ui-btn-right main-logo" href="#popup-welcome"  title="Show Info"></a>
+            </div><!-- /header -->
+            <div  role="main" class="ui-content" id="main-body">
```

Comment:
When I open the page the height to display the tree is very small:
![image](https://user-images.githubusercontent.com/6410125/151213910-c486092d-08ff-4f16-aab2-fbbb5e4f1311.png)
Do you also have this on your side? Maybe it's possible to enforce that the hright gets bigger?
In chromium, when I increase the height of an element with id="jsv-tree" I can get it to full screen